packages/config/config/devices/0x031e/lzw36.json

```diff
@@ -0,0 +1,559 @@
+            "valueSize": 4,
+            "minValue": 0,
+            "maxValue": 100600319,
+            "defaultValue": 0,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "25": {
+            "label": "Fan LED Strip Effect",
+            "description": "Please see website for documentation.\n          Byte 4: Effect - 0 = Off, 1 = Solid, 2 = Slow Blink, 3 = Fast Blink, 4 = Chase, 5 = Pulse\n          Byte 3: Duration - 1 to 60 = seconds, 61 to 120  minutes, 121 -  254 = hours, 255 = Indefinitely\n          Byte 2: Intensity - 0 to 9.  0 = dim, 9 = bright\n          Byte 1: Color - 0 - 255. Hue color wheel. value/255 * 360 = Hue color wheel value\n      Range: 0-100600319\n      Default: 0",
+            "valueSize": 4,
+            "minValue": 0,
+            "maxValue": 100600319,
+            "defaultValue": 0,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "26": {
+            "label": "Light LED Strip Timeout",
+            "description": "When the LED strip is disabled (Light LED Strip Intensity is set to 0), this setting allows the LED strip to turn on temporarily while being adjusted.\n      Range: 0-10\n      Default: 3",
+            "valueSize": 1,
+            "minValue": 0,
+            "maxValue": 10,
+            "defaultValue": 3,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": false,
+            "options": [
+                {
+                    "label": "Stay Off",
+                    "value": 0
+                },
+                {
+                    "label": "1 Second",
+                    "value": 1
+                },
+                {
+                    "label": "2 Seconds",
+                    "value": 2
+                },
+                {
+                    "label": "3 Seconds",
+                    "value": 3
+                },
+                {
+                    "label": "4 Seconds",
+                    "value": 4
+                },
+                {
+                    "label": "5 Seconds",
+                    "value": 5
+                },
+                {
+                    "label": "6 Seconds",
+                    "value": 6
+                },
+                {
+                    "label": "7 Seconds",
+                    "value": 7
+                },
+                {
+                    "label": "8 Seconds",
+                    "value": 8
+                },
+                {
+                    "label": "9 Seconds",
+                    "value": 9
+                },
+                {
+                    "label": "10 Seconds",
+                    "value": 10
+                }
```

Comment:
These options seem pretty unnecessary. I'd remove them and change `allowManualEntry` to true. You could mention that 0 = off in the description.